_data/modules.yml

```diff
@@ -58,6 +58,10 @@
   description: Codeception FlySystem module with several adapters. Provides a set of methods for checking and modifying files on remote storage.
   color: "#FFF5C4" 
 
+- name: GherkinParam
+  link: http://github.com/edno/codeception-gherkin-param
+  description: The Codeception extension for supporting parameter notation in Gherkin scenario.
```

Comment:
This line still says that it is an extension.

Reply:
Damned. I did not notice that it was also mentioned in the description